must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -156,3 +156,16 @@ echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${B
 echo "collecting dump of oc get volumereplicationgroups all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplicationgroups --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vrg_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect vrg --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# Collect details of storageclassclaim for managed services
+ echo "collecting dump of oc get storageclassclaim" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+namespaces=$(oc get storageclassclaim -A --no-headers | awk '{print $1}')
+for ns in "${namespaces}"; do
```

Comment:
If there are more than one storageclassclaim in a particular namespace, the output file will be appended with duplicate content.

Reply:
Thanks for notifying @jilju, will update

Reply:
you need to remove duplicate namespace entries. or use same way we do for other resources like VR,VSC etc

Reply:
yes probably, checking that